main.go

```diff
@@ -663,8 +740,14 @@ func configureSite(ctx *cli.Context, siteNum int, siteStrs []string, healthCheck
 
 		proxy := &httputil.ReverseProxy{
 			Director: func(r *http.Request) {
-				r.Header.Add("X-Forwarded-Host", r.Host)
-				r.Header.Add("X-Real-IP", r.RemoteAddr)
+				if strings.HasPrefix(r.RemoteAddr, "localhost") || strings.HasPrefix(r.RemoteAddr, "127.0.0.1") {
+					r.Header.Add("X-Forwarded-Host", getHost(r))
+					r.Header.Add("X-Real-IP", getRemoteAddr(r))
```

Comment:
These two values also must also be globalized and cached, since they do not change for an instance of the sidekick.

Reply:
Ack